.github/workflows/ci.yml

```
@@ -1,11 +1,35 @@
+
+yaml
+
 name: CI
```

actionlint (1.7.12)

[error] could not parse as YAML: mapping values are not allowed in this context

(syntax-check)